### ISSUE TITLE
feat: set standard theme accessibility to AA standard

### ISSUE THREAD

**Issue by jaymirecki** [outoforbitdev/library-react-core#48]
Currently we attempt to reach AAA accessibility contrast ratios. While a high contrast is helpful for users, AAA results in a washed out color palette. Instead, we should target AA standards. This means that the following combinations should have the following contrast ratios:

| Foreground | Background | Contrast Ratio |
|-------------|-------------|---------------|
| primary-text | primary-background | 4.5:1 |
| primary-link | primary-background | 4.5:1 |
| primary-link-visited | primary-background | 4.5:1 |
| primary-text | primary-shade | 4.5:1 |
| primary-link | primary-shade | 4.5:1 |
| primary-link-visited | primary-shade | 4.5:1 |
| error-text | primary-background | 4.5:1 |
| warning-text | primary-background | 4.5:1 |
| submit-text | primary-background | 4.5:1 |
| accent-text | primary-background | 4.5:1 |
| primary-text              | primary-link | 3:1 |
| primary-text              | primary-link-visited | 3:1 |
| secondary-text | secondary-background | 4.5:1 |
| secondary-link | secondary-background | 4.5:1 |
| secondary-link-visited | secondary-background | 4.5:1 |
| secondary-text | secondary-shade | 4.5:1 |
| secondary-link | secondary-shade | 4.5:1 |
| secondary-link-visited | secondary-shade | 4.5:1 |
| error-text | secondary-background | 4.5:1 |
| warning-text | secondary-background | 4.5:1 |
| submit-text | secondary-background | 4.5:1 |
| accent-text | secondary-background | 4.5:1 |
| secondary-text              | secondary-link | 3:1 |
| secondary-text              | secondary-link-visited | 3:1 |
| accent-block-text | accent-block-background | 4.5:1 |
| accent-block-link | accent-block-background | 4.5:1 |
| accent-block-link-visited | accent-block-background | 4.5:1 |
| accent-block-text | accent-block-shade | 4.5:1 |
| accent-block-link | accent-block-shade | 4.5:1 |
| accent-block-link-visited | accent-block-shade | 4.5:1 |
| accent-block-text              | accent-block-link | 3:1 |
| accent-block-text              | accent-block-link-visited | 3:1 |
| error-block-text | error-block-background | 4.5:1 |
| error-block-text | error-block-shade | 4.5:1 |
| warning-block-text | warning-block-background | 4.5:1 |
| warning-block-text | warning-block-shade | 4.5:1 |
| submit-block-text | submit-block-background | 4.5:1 |
| submit-block-text | submit-block-shade | 4.5:1 |
